src/common.ts

```diff
@@ -18,6 +18,7 @@ export interface RequestOptions {
   jwt?: string;
   headers?: Headers;
   body?: any;
+  qs?: object;
```

Comment:
what does `qs` stand for? Query String? I think we should use the full name instead